### ISSUE TITLE
Commenting Confusion

### ISSUE THREAD

**Issue by rkillackey** [uchicago-codes/MPCS50101-2017-Winter-Forum#44]
Hi everyone,

I wanted to post a quick message just to clear up any confusion about leaving comments on your homework problems. We've heard from a few students that they were unsure about whether comments should be left in the code before submitting assignments. 

To be clear, concise comments describing the functionality of a program, particular function, or variables are encouraged! If you're stuck on a problem, definitely leave comments about your code logic up to that point, and what you would have liked to carry out in code -- the graders will try to be fair in assessing those comments along with any code you have written. 

However, please consider the readability of your assignment when making comments. Comments that are too excessive (over more than a few lines) aren't necessary, and make your assignments hard for the graders to read.

Additionally, commented out _lines of code_, such as `print` statements, or code you're no longer using in your program, _should_ be removed before submitting your assignment (unless they are part of an explanation about why the program doesn't work fully). 

TL;DR: 

- Remove this code: `# print "commented out print statement i'm no longer using"` or this code:
```
# def function_im_not_using():
#     return 'hello'
```

- Leave this code in: `# This is a comment about what the following function does`

Hope this clears things up! Let us know if you have any more questions about leaving comments.
